javalib/src/main/scala/java/lang/process/UnixProcess.scala

```diff
@@ -46,7 +46,7 @@ private[process] object UnixProcess {
       false
     } else if (LinktimeInfo.isLinux) {
       LinuxOsSpecific.hasPidfdOpen()
-    } else if ((LinktimeInfo.isMac) || (LinktimeInfo.isFreeBSD)) {
+    } else if (LinktimeInfo.isMacOrFreeBSD) {
```

Comment:
One could spin off an Issue stating that similar
code could be made available on Linux with a
feature test similar to the `LinuxOsSpecific.hasPidfdOpen`
just above.

Of course, macOS & BSD kin could also do such a 
feature test for the Open Group Name. If that failed,
it could check for the '_np' name.

The 1++ proliferate faster than rabbits.